### ISSUE TITLE
Build on pull request review submission

### ISSUE THREAD

**Issue by yubo56** [jenkinsci/ghprb-plugin#622]
https://github.com/jenkinsci/ghprb-plugin/blob/master/src/main/java/org/jenkinsci/plugins/ghprb/GhprbRootAction.java#L140 - looks like we only trigger right now on issue_comment or pull_request, which according to https://developer.github.com/webhooks/ will not be triggered when a pull request review is submitted. Is there any chance this could be supported? Think changing https://github.com/jenkinsci/ghprb-plugin/blob/master/src/main/java/org/jenkinsci/plugins/ghprb/GhprbRootAction.java#L161 to be a "contains" instead of "equalsIgnoreCase" or adding another line would accomplish the same thing

Lets CI that fetches review comments etc. rerun when reviews are updated

**Comment by yubo56** [comment]
```Request not known for event: pull_request_review``` see this in the jenkins logs

**Comment by samrocketman** [comment]
@yubo56 question: what does building on pull request review accomplish?  I'm having a hard time understanding why that would be desired.  Can you elaborate?

**Comment by yubo56** [comment]
We have a CI system that currently computes whether a PR is approved by looking at comments containing `lgtm`, but we want to move to approved PR reviews to line up more with other projects at our organization. Thus, we want approval to be recomputed every time a new PR review is submitted, the same trigger that an `issue_comment` would generate.

It's a pretty esoteric (and arguably bad) use case, but I can't imagine a scenario where allowing the user to pass in such events would be a hindrance? Users that don't want to trigger on PR review can just disable the webhook event anyways, seems like a feature that puts more power in the hands of the user.

**Comment by samrocketman** [comment]
I understand the use case now.  Basically, you want to replace "lgtm" comments with an approved review which does the same thing.

**Comment by yubo56** [comment]
Yup, same thing i.e. retrigger our approval checking job. Thanks, I'll stay tuned for if/when this gets added

**Comment by dougalg** [comment]
I also would like to see something like this. My idea workflow would be to run the job (a) on approval and (b) on re-test comment IF the PR has been approved. This way we can do automated merging if the tests, etc. all pass in one pipeline.

**Comment by apatr** [comment]
@samrocketman We have the exact same use case and would like to trigger PRs based on Lead approving a PR 
Currently , we use ghprb plugin to trigger builds based on issue_comment "build"
Any idea when this can be expected? We would like to avoid writing custom scripts just for this case.

**Comment by samrocketman** [comment]
@apatr in general, I don't write a lot of code.  Mostly, I manage pull requests, test releases, and release new revisions of plugins.  This is how I typically handle other plugins I maintain as well.

I highly encourage anyone to dig into this issue which has an explicit need for it.  Contributing code is a great way to raise your own profile as well as a company in the community.  I encourage you to consider investing time in this feature rather than custom scripts.

That's not to say I don't code, just not here at the moment.  I will not likely implement this myself at this time.

**Comment by samrocketman** [comment]
@apatr in general we accept pull requests.  So if your pull request works, has tests, and does not break plugin upgrading then it is typically accepted.

**Comment by sodhia17** [comment]
@samrocketman were you able to make the changes and initiate the PR for the same. If yes, could you share your PR number ?

**Comment by nitinsurana** [comment]
Here's the PR for adding pull_request_review_comment webhook to trigger builds - https://github.com/jenkinsci/github-branch-source-plugin/pull/185
